files/galaxy/dynamic_rules/usegalaxy/dexseq.py

```diff
@@ -1,34 +0,0 @@
-from galaxy.jobs import JobDestination
-import os
-
-def dexseq_memory_mapper( job, tool ):
```

Comment:
@kysrpex have we taken this magic over to TPV?

Reply:
We have not. It was apparently not used by sorting hat (see [66ec444](https://github.com/usegalaxy-eu/infrastructure-playbook/commit/66ec444277f575fcc5651c1158e0e191eca55bb9)).